soc/arm/nxp_imx/rt/power_rt10xx.c

```diff
@@ -76,6 +76,13 @@ static void lpm_set_sleep_mode_config(clock_mode_t mode)
 	GPC_DisableIRQ(GPC, GPR_IRQ_IRQn);
 }
 
+static void lpm_enter_soft_off_mode(void)
```

Comment:
Hi @NickolasLapp, thank you for the contribution here. I think enabling SNVS soft off mode for the RT series would be valuable, but I am concerned about doing so without a simple sample application (something like `samples/boards/nrf/system_off`.

Without a sample application, testing this contribution becomes difficult, and the value to the community is lower, since users don't have a demonstration of how to enter SNVS mode on their RT board without first writing their own Zephyr application. Beyond this, the GPC supports a variety of wakeup sources, and adding a sample application would ensure that the RTC was always set up with an alarm, as well as help users understand the relationship between the GPC api call here and the setup required before entering low power mode.